keps/sig-windows/20190714-windows-csi-support.md

```diff
@@ -44,11 +44,17 @@ superseded-by:
     - [Enhancements in Kubelet Plugin Watcher](#enhancements-in-kubelet-plugin-watcher)
     - [Enhancements in CSI Node Driver Registrar](#enhancements-in-csi-node-driver-registrar)
     - [New Component: CSI Proxy](#new-component-csi-proxy)
+      - [CSI Proxy Named Pipes](#csi-proxy-named-pipes)
+      - [CSI Proxy Configuration](#csi-proxy-configuration)
+      - [CSI Proxy GRPC API](#csi-proxy-grpc-api)
+      - [CSI Proxy GRPC API Graduation and Deprecation Policy](#csi-proxy-grpc-api-graduation-and-deprecation-policy)
+      - [CSI Proxy Event Logs](#csi-proxy-event-logs)
     - [Enhancements in Kubernetes/Utils/mounter](#enhancements-in-kubernetesutilsmounter)
     - [Enhancements in CSI Node Plugins](#enhancements-in-csi-node-plugins)
```

Comment:
nit: Could you define CSI Node Plugins? Maybe in a glossary section? I wasn't sure if this is referring to the CSI driver to be installed on nodes or the DaemonSet or something else without reading deeper